README.md

```diff
@@ -0,0 +1,72 @@
+# Flatpak package for Xilinx Vivado Design Suite
```

Comment:
Most, if not all of this info should be in the appdata. Users are unlikely to ever come here.

Reply:
Ok, I will add it